spec/controllers/items_controller_spec.rb

```diff
@@ -24,6 +24,8 @@
   end
 
   describe 'index' do
+    # rubycritic dislikes this, but we disagree because we need this test
```

Comment:
Since we don't evaluate spec files with rubycritic, comments justifying code are not really necessary. Should be fine to merge, though.